libs/shared/console-shared/src/lib/cluster-settings/ui/cluster-general-settings/cluster-general-settings.tsx

```diff
@@ -23,7 +23,12 @@ export function ClusterGeneralSettings(props: ClusterGeneralSettingsProps) {
             className="mb-3"
             dataTestId="input-name"
             name={field.name}
-            onChange={field.onChange}
+            onChange={(e) => {
+              if (e.target.value.toLowerCase().includes('prod')) {
```

Comment:
Do we want the condition in the detail view as well ? I thought it was for creation flow only

Reply:
I can limit it to creation flow
https://github.com/Qovery/console/pull/1680/commits/b968d9f91d555e46f8f9d271aadf90bfdd2c6d57